src/olympia/devhub/templates/devhub/new-landing/components/overview.html

```diff
@@ -2,12 +2,12 @@
   <div>
     <h1>{{ _('Customize Your Firefox') }}</h1>
     <p>
-      {% trans %}
+      {% trans html_link="https://developer.mozilla.org/docs/Web/HTML", js_link="https://developer.mozilla.org/docs/Web/JavaScript", css_link="https://developer.mozilla.org/docs/Web/CSS" %}
       Add-ons let millions of Firefox users enhance their browsing experience.
-      If you know <a href="#">HTML</a>, <a href="#">JavaScript</a>, and <a href="#">CSS</a>,
+      If you know <a href="{{ html_link }}">HTML</a>, <a href="{{ js_link }}">JavaScript</a>, and <a href="{{ css_link }}">CSS</a>,
```

Comment:
Since afaict HTML, JavaScript and CSS don't require translating, the vars could contain the entirety of `<a href=""...></a>` that would still allow the translator to define the word order.